pages/news/nightstand.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Nightstand mode
+layout: news
+posted: 2023-03-08
+post: true
+thumbnails: nightstand
+summary: The new Nighstand mode shows the state of a charging watch at a glance.
+author: Ed Beroset (beroset)
+---
+
+<p>When a watch is on the charger, it's useful to be able to see the state of charge at a glance.  We've implemented a new feature called *Nightstand mode* that helps with this.</p>
+
+<p>As is often the case with AsteroidOS development, there is often conversation about what new features should be implemented and how they should work.  To help explain to the rest of the community what Nightstand mode might do, and how it might work, the following scenario and requirements descriptions were inserted into a <a href="https://github.com/AsteroidOS/asteroid/issues/151">issue description</a>.</p>
+
+<h2>Scenario</h2>
+
+<p>A tired traveler arrives at a hotel late at night and finds there is no clock in the room.  Fortunately, this person has an AsteroidOS watch with nightstand mode.  She uses the settings to select the kind of display that she would like to use for this and enables the mode.  Then she puts the watch on its charging stand where it acts as sort of a bedside clock, prominently featuring the state of charge and the current time.  She wears glasses and does not want to have to put them on to see the time, so she chooses a watchface that is high contrast and features large numbers for the time.  Even with that, it's not always that easy to see the time, so it is configured so that at 3AM with no glasses, she can pick up the watch and bring it closer to note the time (ugh!  3AM!) and replace it on the stand within some configurable period of time with the watch not leaving nightstand mode.
```

Comment:
Keep the scenario which is entertaining to read. Maybe you could quote it in the style of eltmoset's article on gadgetbridge ?

Also, it looks like it's missing a `</p>` ? probably it'll be easier once written in Markdown :)

Reply:
I started in markdown, but our news articles do not support it at the moment and I didn't take the time to dig into that yet.  Also, I started by using `<blockquote>` but that caused the text to be light gray and didn't look good.  I didn't have tome to dive into the templates at the moment.

Reply:
Ugh. I see :( then forget about Markdown for now ...

Reply:
Fixed!  Merge #280 first and we'll be able to use this updated markdown version.